management/client_test.go

```diff
@@ -87,6 +87,17 @@ func TestClient_List(t *testing.T) {
 	assert.Contains(t, clientList.Clients, &Client{ClientID: expectedClient.ClientID})
 }
 
+func TestClient_ListAll(t *testing.T) {
+	configureHTTPTestRecordings(t)
```

Comment:
This doesn't really test the pagination aspect that well, it might be best to allow overriding the PerPage so that we can set it to a low value here and properly test that pagination aspect

Reply:
Should we add a test similar to https://github.com/auth0/auth0-cli/blob/7d27438692215cc4904f5116e1a3e82a030d25a9/internal/cli/terraform_fetcher_test.go#L43 ?